### PR TITLE
Migrate to github actions

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Migrate from travis to GitHub actions to bring this repo in-line with eq-questionnaire-runner, eq-questionnaire-validator etc.

### How to review 
Check the action tests pass.

### CODE REVIEW

**Comment by berroar** [comment]
[Migrate eq-translations from Travis to Github actions (S/M)](https://trello.com/c/zjU8di1U/3750-migrate-eq-translations-from-travis-to-github-actions-s-m)

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: PR
```
Comment (MebinAbraham) [inline .github/workflows/pull_request.yml]: This is for PRs, would also need one to push to Docker on merge.

Comment (berroar) [reply .github/workflows/pull_request.yml]: I've added another action

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+on:
+  pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
```
Comment (JamesGardiner) [inline .github/workflows/pull_request.yml]: Should be pinned

File: .github/workflows/pull_request.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        run: pipenv install --dev
+      - name: Security Check
+        run: pipenv check ./eq_translations ./tests
+      - name: Lint Python
+        run: pipenv run pylint eq_translations ./tests
+      - name: Black
+        run: pipenv run black --check .
+  test:
+    needs: lint
+    runs-on: ubuntu-latest
```
Comment (JamesGardiner) [inline .github/workflows/pull_request.yml]: Pin this

**Comment by JamesGardiner** [comment]
Looks good - just needs inning to the 18.04 LTS Ubuntu so we don't get upgraded unwittingly

**Pull request merged**